titan-web-client/package.json

```diff
@@ -132,5 +132,6 @@
     "presets": [
       "react-app"
     ]
-  }
+  },
+  "proxy": "http://[::1]:8000"
```

Comment:
The ipv6 address isn't playing well with my linux environment. I'll have to look into it. Is there a reason why you chose this over ipv4?

Reply:
IPv4 doesn't play well on the Mac as far as I can tell. I'll test it again and see if we can just bind to `127.0.0.1`.